diff -Nur -x '*.orig' -x '*.rej' e-smith-runit-1.0.0/root/etc/rc.d/init.d/daemontools mezzanine_patched_e-smith-runit-1.0.0/root/etc/rc.d/init.d/daemontools
--- e-smith-runit-1.0.0/root/etc/rc.d/init.d/daemontools	2008-03-26 14:32:35.000000000 -0600
+++ mezzanine_patched_e-smith-runit-1.0.0/root/etc/rc.d/init.d/daemontools	2008-03-26 14:32:16.000000000 -0600
@@ -10,8 +10,8 @@
 
 # Determine the service name and its service directory from $0
 
-SERVICE=$(/bin/basename $0 | sed -e 's/^[SK][0-9][0-9]*//')
-SERVICE_DIR=/service/$SERVICE
+prog=$(/bin/basename $0 | sed -e 's/^[SK][0-9][0-9]*//')
+SERVICE_DIR=/service/$prog
 
 ###########################################################################
 
@@ -24,28 +24,28 @@
 
 start()
 {
-    /bin/echo -n "Starting $SERVICE:"
+    /bin/echo -n $"Starting $prog:"
     dirs=$1
     if [ -d $1/log ]; then
         dirs="$1/log $1"
     fi
     /usr/bin/sv u $dirs > /dev/null
     if [ $? -ne 0 ]; then
-        failure "Starting $SERVICE"
+        failure "Starting $prog"
     else
-        success "Starting $SERVICE"
+        success "Starting $prog"
     fi
     /bin/echo
 }
 
 stop()
 {
-    /bin/echo -n "Stopping $SERVICE:"
+    /bin/echo -n $"Stopping $prog:"
     /usr/bin/sv stop $1 > /dev/null
     if [ $? -ne 0 ]; then
-        failure "Stopping $SERVICE"
+        failure "Stopping $prog"
     else
-        success "Stopping $SERVICE"
+        success "Stopping $prog"
     fi
     /bin/echo
 }
@@ -55,7 +55,7 @@
 # uninstall of the service in question.
 svdisable()
 {
-    /bin/echo -n "Disabling $SERVICE:"
+    /bin/echo -n $"Disabling $prog:"
     stop $1
     cd $1 && rm -f $1
     dirs=.
@@ -65,9 +65,9 @@
     /usr/bin/sv d $dirs
     /usr/bin/sv x $dirs
     if [ $? -ne 0 ]; then
-        failure "Disabling $SERVICE"
+        failure "Disabling $prog"
     else
-        success "Disabling $SERVICE"
+        success "Disabling $prog"
     fi
     /bin/echo
 }
@@ -75,44 +75,44 @@
 case $1 in
 
     restart)
-	action "Restarting $SERVICE" /usr/bin/sv t $SERVICE_DIR
+	action "Restarting $prog" /usr/bin/sv t $SERVICE_DIR
 	/usr/bin/sv u $SERVICE_DIR
             ;;
 
     sigalrm)
-	action "Sending ALRM signal to $SERVICE" /usr/bin/sv a $SERVICE_DIR
+	action "Sending ALRM signal to $prog" /usr/bin/sv a $SERVICE_DIR
             ;;
 
     sigcont)
-	action "Sending CONT signal to $SERVICE" /usr/bin/sv c $SERVICE_DIR
+	action "Sending CONT signal to $prog" /usr/bin/sv c $SERVICE_DIR
             ;;
 
     sighup)
-	action "Sending HUP signal to $SERVICE" /usr/bin/sv h $SERVICE_DIR
+	action "Sending HUP signal to $prog" /usr/bin/sv h $SERVICE_DIR
             ;;
 
     sigusr1)
-	action "Sending USR1 signal to $SERVICE" /usr/bin/sv 1 $SERVICE_DIR
+	action "Sending USR1 signal to $prog" /usr/bin/sv 1 $SERVICE_DIR
             ;;
 
     sigusr2)
-	action "Sending USR2 signal to $SERVICE" /usr/bin/sv 2 $SERVICE_DIR
+	action "Sending USR2 signal to $prog" /usr/bin/sv 2 $SERVICE_DIR
             ;;
 
     sigint)
-	action "Sending INT signal to $SERVICE" /usr/bin/sv i $SERVICE_DIR
+	action "Sending INT signal to $prog" /usr/bin/sv i $SERVICE_DIR
             ;;
 
     sigkill)
-	action "Sending KILL signal to $SERVICE" /usr/bin/sv k $SERVICE_DIR
+	action "Sending KILL signal to $prog" /usr/bin/sv k $SERVICE_DIR
             ;;
 
     sigstop)
-	action "Sending STOP signal to $SERVICE" /usr/bin/sv p $SERVICE_DIR
+	action "Sending STOP signal to $prog" /usr/bin/sv p $SERVICE_DIR
             ;;
 
     sigterm|condrestart)
-	action "Sending TERM signal to $SERVICE" /usr/bin/sv t $SERVICE_DIR
+	action "Sending TERM signal to $prog" /usr/bin/sv t $SERVICE_DIR
             ;;
 
     status)
@@ -132,7 +132,7 @@
             ;;
 
     *)
-        echo "usage: $0 {start|stop|restart|status|sigalrm|sigcont|sighup|sigint|sigkill|sigstop|sigterm|condrestart|sigusr1|sigusr2|svdisable}"
+        echo $"usage: $0 {start|stop|restart|status|sigalrm|sigcont|sighup|sigint|sigkill|sigstop|sigterm|condrestart|sigusr1|sigusr2|svdisable}"
         ;;
 
 esac
